### DISCUSSION TITLE
Why would this happen?!

### DISCUSSION THREAD

**Discussion by Navid200** [NightscoutFoundation/xDrip#2070] [Q&A]
The user claims he used "Add Calibration" to enter the calibration.
But, why would the transmitter make a correction in the opposite direction?

![277765895_5002170656564330_2567401296737674158_n](https://user-images.githubusercontent.com/51497406/162523528-d55ae376-dc0e-4865-9c65-19b6948f548e.jpg)
![277788897_5001707016610694_7998428820219075980_n](https://user-images.githubusercontent.com/51497406/162523574-738ea4ac-2852-4f1e-87cc-38dab734442b.jpg)
![cal](https://user-images.githubusercontent.com/51497406/162523611-d5893793-8bed-43b0-9cb7-e662cbc19407.jpg)

**Comment by Navid200** [comment]
@joshbohmer

**Comment by tzachi-dar** [comment]
One reason that this would happen in the old g4 days is this:
Calibration was based on the last 6 measurements. When you enter a calibration you would also remove a calibration.
Now let's assume that the last calibration was pulling things down. Now, you enter a calibration that moves things down (but less than the last one), so, the overall effect is a move up.
Now, this was something that happened in the G4 days, and algorithms have changed. Still, I think that if one calibration pushes the other calibration back (for example some time decade) it can cause this effect.
